client/src/app/site/motions/components/motion-detail/motion-detail.component.ts

```diff
@@ -206,6 +213,20 @@ export class MotionDetailComponent extends BaseComponent implements OnInit {
         });
     }
 
+    /**
+     * returns all Possible supporters
+     */
+    public getPossibleSupporters(): User[] {
```

Comment:
Seems you could use just one function, since they both just execute `this.DS.getAll(User);`
(getUsers() or ssimilar)

Reply:
Shouldn't it be a function, so one can maybe add a filter or some other mechanisms?

Reply:
I would suggest making a `getAllUsers` as long as anyone can be a submitter and supporter and using this functions for both lists.